src/services/helperService.ts

```
@@ -36,4 +36,45 @@ export const formatDate = (value: string) => {
     month: "short",
     year: "numeric",
   });
-};
\ No newline at end of file
+};
+
+export const convertKeysToTitleCase = (obj: Record<string, any>): Record<string, any> => {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)